.circleci/config.yml

```diff
@@ -272,6 +315,10 @@ workflows:
           image_tag: heroku/pack:20-build
           image_tag_alias: heroku/pack:20-build
           requires:
+            - test-go-20
+            - test-java-20
+            - test-php-20
+            - test-python-20
             - create-20-build-image
```

Comment:
The `test-*` tasks themselves transitively require the `create-20-build-image` task, so it can be removed here:
```suggestion
```

Alternatively if we wanted to be explicit about the dependency, we should add it to the 18 equivalent of this step, for consistency.